8.9.2 Release notes (#3820)
elastic/security-docs: docs/release-notes/8.9.asciidoc

[[release-notes-header-8.9.0]]
== 8.9

[discrete]
[[release-notes-8.9.2]]
=== 8.9.2

[discrete]
[[bug-fixes-8.9.2]]
==== Bug fixes

> * Fixes legacy showTopN popover actions ({pull}165099[#165099]).

Author: @PhilippeOberti @christineweng should this be release-noted for 8.9.2? It looks like a fix for the expandable alert details flyout, which is being introduced in 8.10.

cc: @semd

Author: Confirmed with @PhilippeOberti that this isn't specific to the new expandable flyout, so I'll be keeping it in unless directed otherwise.

Author: ```suggestion
* Fixes a bug that stopped the additional inline actions from completing ({pull}165099[#165099]).
```

* Fixes a blocklist bug that prevented file path entries for Windows and macOS applications from being passed as case insensitive ({pull}164200[#164200]).

> * Modal says "duplicating 0 rules" when you duplicate an individual rule ({pull}163908[#163908]).

Author: @banderror what should the release note summary for this bug fix be?

Reviewer: @nastasha-solomon Could be something like "Fixes a message in the confirmation modal window showed when duplicating a rule"

Author: ```suggestion
* Fixes a bug in the confirmation message that appears when you're duplicating a single rule ({pull}163908[#163908]).
```

> * Fixes a bug in the confirmation message that appears when you're duplicating a single rule ({pull}163908[#163908]).

Author: ```suggestion
* Fixes a bug in the confirmation message that appears when you're duplicating a single rule ({pull}163908[#163908]).
* Fixes a bug on the rule details page that showed the Data view label twice if you were viewing a rule using a data view.  
```

[[release-notes-header-8.9.0]]
== 8.9

[discrete]
[[release-notes-8.9.2]]
=== 8.9.2

[discrete]
[[bug-fixes-8.9.2]]
==== Bug fixes

> * Fixes a bug that stopped the additional inline actions from completing ({pull}165099[#165099]).

Reviewer 2: Correct me if I'm wrong re: the location. 
```suggestion
* Fixes a bug that prevented inline actions on the Alerts page from completing ({pull}165099[#165099]).
```

Author: The example in the PR only showed the Alerts page, so I'm going to assume your change is correct. @semd would you mind verifying?

Reviewer 3: Yes, the legacy actions that were fixed are only rendered with the "row-renderer" component, which is used to render the `reason` field. It is displayed in the flyout, and in the alerts table as well, so yes, both are on the Alerts page.

Author: Thanks for confirming! I'll merge this change then.

> * Fixes a blocklist bug that prevented file path entries for Windows and macOS applications from being passed as case insensitive ({pull}164200[#164200]).

Reviewer 2: ```suggestion
* Fixes a bug that prevented blocklist file path entries for Windows and macOS applications from being passed as case insensitive ({pull}164200[#164200]).
```

> * Fixes a bug in the confirmation message that appears when you're duplicating a single rule ({pull}163908[#163908]).

Reviewer 2: ```suggestion
* Fixes a bug in the confirmation message that appears when duplicating a single rule ({pull}163908[#163908]).
```

> * Fixes a bug on the rule details page that showed the Data view label twice if you were viewing a rule using a data view.  

Author: ```suggestion
* Fixes a bug on the rule details page that showed the Data view label twice if you were viewing a rule using a data view ({pull}164494[#164494]).
```

> * Fixes a bug that affected Timeline when you investigated an alert that was created from a rule with exceptions ({pull}162190[#162190]).

Reviewer 2: ```suggestion
* Fixes a bug that affected Timeline when you investigated an alert created from a rule with exceptions ({pull}162190[#162190]).
```
